packages/pharos/src/components/tabs/pharos-tabs.ts

```diff
@@ -66,28 +64,29 @@ export class PharosTabs extends PharosElement {
     return this.querySelector(`#${panelId}`);
   }
 
-  private _handleTabSelected(event: Event): void {
-    const selected = event.target as PharosTab;
-    const previous: PharosTab | null = this.querySelector(
-      `[data-pharos-component="PharosTab"][selected]:not([id="${selected.id}"])`
+  private _handleTabSelected(selectedTab: PharosTab): void {
+    selectedTab.selected = true;
+
+    const previousTab: PharosTab | null = this.querySelector(
+      `${_allTabsSelector}[selected]:not([id="${selectedTab.id}"])`
     );
 
-    if (previous) {
-      previous.selected = false;
-      const panel: PharosTabPanel | null = this.querySelector(
-        `[data-pharos-component="PharosTabPanel"][id="${previous.getAttribute('aria-controls')}"]`
+    if (previousTab && previousTab != selectedTab) {
```

Comment:
There was a bug where you could click a tab repeatedly and it would select/deselect/select/deselect.